recipes/tf_slim/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "TF-slim" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/Maxar-Corp/ratpac/releases/download/slim-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cca9308a93f99331641febbe53c19e8fdf845306250df0cf07ac340b5e0deda9
+
+build:
+  number: 0
+  skip: true  # [win]  # Skip all windows for now due to numpy/scipy version pinning issues
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - tensorflow >=1.12.0
+
+test:
+  imports:
+    # From tensorflow-feedstock:
+    # "Skip the import test on Linux as wheel file require a more recent
+    # version of GLIBC++ than the VM used to build and test package.
+    # Also skip on OSX due to OSX version requirements, see
+    # https://github.com/conda-forge/tensorflow-feedstock/pull/62"
+    - tf_slim  # [win]
+  requires:
+    - tensorflow >=1.12.0
+
+about:
+  home: https://github.com/Maxar-Corp/ratpac/tree/tf_slim/TF-slim
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: ' TF-slim is a lightweight high-level API of TensorFlow for defining, training and evaluating complex models'
+
+  description: |
+    TF-slim is a new lightweight high-level API of TensorFlow
+    for defining, training and evaluating complex models. This
+    directory contains code for training and evaluating several
+    widely used Convolutional Neural Network (CNN) image
+    classification models using TF-slim.
+
+
+extra:
+  recipe-maintainers:
+    - adbeda
+    - agcopenhaver
+    - tata17
```

Comment:
@agcopenhaver @tata17 are you OK with being maintainers here?

Reply:
@scopatz we're on the same team. i thought they could help on this in the future as well.

Reply:
yep that makes sense, but we still need their explicit approval

Reply:
@scopatz Good with me being a maintainer.

Reply:
@scopatz happy to be a maintainer